rpcs3/Emu/System.cpp

```diff
@@ -546,14 +566,20 @@ bool Emulator::BootGame(const std::string& path, const std::string& title_id, bo
 				if (fs::is_file(elf))
 				{
 					m_path = elf;
-					Load(title_id, add_only, force_global_config);
-					success = true;
+					found_any = true;
+					if (last_status) // keep no_errors if even one load was successful
```

Comment:
we only update boot_error if it was an error already
```suggestion
					if (const auto err = Load(title_id, add_only, force_global_config); err != game_boot_result::no_errors)
					{
						result = err;
					}
```